pandas/core/arrays/period.py

```diff
@@ -350,13 +350,10 @@ def to_timestamp(self, freq=None, how='start'):
     # --------------------------------------------------------------------
     # Array-like / EA-Interface Methods
 
-    def __repr__(self):
-        return '<{}>\n{}\nLength: {}, dtype: {}'.format(
-            self.__class__.__name__,
-            [str(s) for s in self],
-            len(self),
-            self.dtype
-        )
+    def _formatter(self, formatter=None):
+        if formatter:
+            return formatter.formatter or str
+        return "'{}'".format
```

Comment:
why not `str` like on the line above?

Reply:
This was to support quoting the periods in the array repr but not in the Series. If we want to quote in both places (or not) then we can have just a single one.

PeriodIndex does quote.

Reply:
https://github.com/pandas-dev/pandas/pull/23601#issuecomment-439882626

Reply:
is this the same as what DatetimeArray / Timedeltarray will do?